src/main/java/blackjack/domain/Participant.java

```diff
@@ -0,0 +1,49 @@
+package blackjack.domain;
+
+import java.util.ArrayList;
+
+abstract public class Participant {
+
+    private static final String COLON = " : ";
+
+    protected final String name;
+    protected final ParticipantCards cards;
+    protected int winScore, loseScore;
+
+    public Participant(String name, ParticipantCards cards, int winCount, int loseCount) {
+        this.name = name;
+        this.cards = cards;
+        this.winScore = winCount;
+        this.loseScore = loseCount;
+    }
+
+    public Participant(String name) {
+        this(name, new ParticipantCards(new ArrayList<>()), 0, 0);
+    }
+
+    public final void judgeScore(int maxScore) {
+        if (sumCardScore() == maxScore) {
+            winScore++;
+            return;
+        }
+        loseScore++;
+    }
+
+    public final String getName() {
+        return name;
+    }
+
+    public final String holdingInfo() {
+        return name + COLON + cards.toString();
+    }
+
+    public final int sumCardScore() {
+        return cards.sumCardScore();
+    }
+
+    public abstract void drawCardMultiple(Deck deck, int number);
```

Comment:
```suggestion
    public abstract void drawCardMultiple(Deck deck);
```
메서드 이름에서 행위가 표현됩니다. 굳이 카드 숫자를 받아 처리할 필요가 없을 것 같습니다.